testsuite/MDAnalysisTests/transformations/test_nojump.py

```diff
@@ -329,6 +329,19 @@ def test_nojump_constantvel_stride_2(nojump_universes_fromfile):
             pass
 
 
+def test_nojump_raises_valueerror_outside_first_frame(nojump_universe):
+    """
+    Test if NoJump transformation raises ValueError when applied outside the first frame.
+    """
+    match = "Periodic box dimensions not provided at step 0"
+
+    with pytest.raises(ValueError, match=match):
+        u = nojump_universe
+        u.trajectory[-1]  # Move to the last frame
```

Comment:
Maybe just move to the second frame
```python
u.trajectory[1]
```
instead of the last one?

*Maybe* it makes a difference if you add the transformation first and then do the trajectory frame change afterwards?